medical-portal/src/UI/src/app/submission-requirements/submission-requirements.component.html

```diff
@@ -152,13 +152,15 @@
               </div>
               <div style="grid-area: D">
                 <div>
-                  <button
-                    mat-flat-button
-                    class="primaryButton"
-                    (click)="openPopup()"
-                  >
-                    Start DMER
-                  </button>
+                  @if (document.documentType === "DMER") {
```

Comment:
can we use an enum here? I know from the backend the DMER is determined by documentTypeCode equals "001"